src/VisualStudio/Core/Def/Telemetry/VisualStudioWorkspaceTelemetryService.cs

```diff
@@ -32,5 +40,29 @@ protected override ILogger CreateLogger(TelemetrySession telemetrySession)
                 new EtwLogger(_optionsService),
                 new VSTelemetryLogger(telemetrySession),
                 Logger.GetLogger());
+
+        protected override void TelemetrySessionInitialized()
```

Comment:
💡 Better to make this async and let the caller perform fire-and-forget on the call.

Reply:
The caller doesn't know an async call is needed.

Reply:
The work to make the callers async is minimal.

Reply:
My point isn't about the work. It's conceptual - why would the caller care about async? It's essentially a trivial notification.

Reply:
It's a non-public API with exactly one caller. Conceptual provides no value because there is nothing here to abstract.